examples/create-react-app-typescript/yarn.lock

```diff
@@ -0,0 +1,11696 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Can we keep either `yarn.lock` or `package-lock.json` for CRA example? Feels redundant to have both

Reply:
Oh, I am sorry. I didn't realize that there are too. I'll take care of it. I suggest adding once of those in the main folder anyway as well.